cocos/core/builtin/builtin-res-mgr.jsb.ts

```diff
@@ -88,6 +89,17 @@ builtinResMgrProto.get = function (uuid: string) {
     return res || this.getAsset(uuid);
 };
 
+
+builtinResMgrProto.compileBuiltinMaterial = function () {
+    // NOTE: Builtin material should be compiled again after the render pipeline setup
+    for (let i = 0; i < this._materialsToBeCompiled.length; ++i) {
+        const mat = this._materialsToBeCompiled[i];
+        for (let j = 0; j < mat.passes.length; ++j) {
+            mat.passes[j].tryCompile();
+        }
+    }
```

Comment:
Need to set `this. _materialsToBeCompiled.length = 0;` at the end of compileBuiltinMaterial?

Reply:
Sure